### PR TITLE
Update with React-Slick master

### PR DESCRIPTION
**Pull request by AHB**
Updates our fork with the latest

### CODE REVIEW

File: examples/CenterMode.js [hunk 1]
```diff
@@ -28,4 +28,8 @@ export default class CenterMode extends Component {
       </div>
     );
   }
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> 4444fc8a93d40e238b6296b386b33dd7512ca456
```
Comment (afairb) [inline examples/CenterMode.js]: Oof

File: examples/Responsive.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ export default class Responsive extends Component {
       speed: 500,
       slidesToShow: 4,
       slidesToScroll: 4,
+<<<<<<< HEAD
```
Comment (afairb) [inline examples/Responsive.js]: 🔦

File: src/arrows.js [hunk 3]
```diff
@@ -53,28 +57,18 @@ export var NextArrow = React.createClass({
     var nextClasses = {'slick-arrow': true, 'slick-next': true};
     var nextHandler = this.clickHandler.bind(this, {message: 'next'});
 
-    if (!this.props.infinite) {
```
Comment (afairb) [inline src/arrows.js]: Was this not some of our custom stuff?

Comment (AHB) [inline src/arrows.js]: No I don't think so, I definitely didn't add it

File: src/inner-slider.js [hunk 4]
```diff
@@ -122,14 +149,16 @@ export var InnerSlider = React.createClass({
 
     var dots;
 
-    if (this.props.dots === true && this.state.slideCount > this.props.slidesToShow) {
+    if (this.props.dots === true && this.state.slideCount >= this.props.slidesToShow) {
```
Comment (afairb) [inline src/inner-slider.js]: I have a feeling we tweaked this in the fork too

File: src/mixins/helpers.js [hunk 5]
```diff
@@ -3,78 +3,128 @@
...
+    } else {
+      slideWidth = this.getWidth(ReactDOM.findDOMNode(this));
+    }
+
+    const slideHeight = this.getHeight(slickList.querySelector('[data-index="0"]'));
+    const listHeight = slideHeight * props.slidesToShow;
 
     // pause slider if autoplay is set to false
-    if(!props.autoplay)
+    if(props.autoplay) {
```
Comment (afairb) [inline src/mixins/helpers.js]: Is this right/tweaked?

**Comment by afairb** [comment]
Is this merge from master or their last tagged/stable release?

File: src/inner-slider.js (hunk 4 above)
Comment (AHB) [inline src/inner-slider.js]: I think we changed something very similar to this, to hide the arrows - but this is dots

I will check though!

**Comment by AHB** [comment]
master 😱

File: src/mixins/helpers.js (hunk 5 above)
Comment (AHB) [inline src/mixins/helpers.js]: Ah yeh I think they just changed it, so when it updates it will pause/auto scroll depending on the prop

Comment (afairb) [inline src/mixins/helpers.js]: It's a bit confusing on their part as either the comment above it is wrong or the `if` is the wrong way around!

File: src/inner-slider.js (hunk 4 above)
Comment (AHB) [inline src/inner-slider.js]: it functions as expected, so I think we are okay!

File: src/mixins/helpers.js (hunk 5 above)
Comment (AHB) [inline src/mixins/helpers.js]: yeah I have no idea what is going on with that, still works okay.....

**Pull request merged**